test/codelense.test.ts

```diff
@@ -57,6 +57,10 @@ resource "null_resource" "reference_test" {
     assert(successful, "forced indexing not successful");
 
     let lenses = await vscode.commands.executeCommand('vscode.executeCodeLensProvider', doc.uri, 10) as vscode.CodeLens[];
+    if (!lenses) {
+      // Retry this seems a bit flakey
+      lenses = await vscode.commands.executeCommand('vscode.executeCodeLensProvider', doc.uri, 10) as vscode.CodeLens[];
+    }
```

Comment:
![flakeytest](https://user-images.githubusercontent.com/1939288/62567156-86e62600-b882-11e9-9347-cfdd71e159d0.png)

I've seen this test be a bit flakey when testing locally and in the build. While not ideal a retry seems to handle this case (sometimes `lenses` is returned as `undefined`) 

Build example: https://mauvezero.visualstudio.com/vscode-terraform/_build/results?buildId=69